### PR TITLE
One pager interface

### PR DESCRIPTION
**Pull request by sumalreddy17**
Add One Pager Plot class structure and method signatures

Introduces core class architecture for marketing analytics visualization
with comprehensive method signatures for all plot types.

Key components:
- PlotData dataclass for standardized data handling
- MarketingOnePager class with plot generation methods
- Method signatures for 8 distinct visualization types

### CODE REVIEW

**Comment by dhavalpatel624624** [comment]
Let's avoid the use of both pyplot and go. Lets pick one over the other. Also, I see some files have Any datatypes, lets get those specified. Can we also add the  `PlotData`  in the constructor of each individual visualizer files and remove it from underlying methods if it makes sense. Then, make sure you instantiate each of the visualizers and call the functions in `generate_onepager` instead of the functions directly.

File: python/src/robyn/visualization/plot_data.py
Comment (sumane81) [inline python/src/robyn/visualization/plot_data.py:1]: You shouldn't need to define new data entity in visualizer module. Input data for visualizer should directly come from component. For example - ParetoVisualizer should take ParetoResult.

File: python/src/robyn/visualization/onepager_visualizer.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    ):
+        """
+        Initialize OnePager with required results objects.
+        
+        Args:
+            pareto_result: Results from Pareto optimization
+            clustered_result: Optional clustering results
+            adstock: Optional adstock configuration
+        """
+        self.pareto_result = pareto_result
```
Comment (sumalreddy17) [inline python/src/robyn/visualization/onepager_visualizer.py:38]: This can be moved to BaseVisualizer.

**Review by sumane81 — CHANGES_REQUESTED** [review]
Comments added

**Review by dhavalpatel624624 — APPROVED** [review]

**Comment by dhavalpatel624624** [comment]
Looks good, thanks for making all the changes! :)

File: python/src/robyn/visualization/input_visualizer.py
Comment (sumane81) [inline python/src/robyn/visualization/input_visualizer.py:1]: input visualizer is vague name. Can we make it more specific?

Comment (sumalreddy17) [reply python/src/robyn/visualization/input_visualizer.py:1]: How about media_response_visualizer?

File: python/src/robyn/visualization/robyn_visualizer.py
Comment (sumane81) [inline python/src/robyn/visualization/robyn_visualizer.py:1]: What is this file for? Can you delete it if not needed?

Comment (sumalreddy17) [reply python/src/robyn/visualization/robyn_visualizer.py:1]: Offline decision to clean up in a different PR

**Review by sumane81 — APPROVED** [review]
Some clean up is needed but that will come in separate PR.

**Pull request merged**